bigquery/cloud-client/requirements.txt

```diff
@@ -1,5 +1,6 @@
 google-cloud-bigquery[pandas]==1.20.0
 google-auth-oauthlib==0.4.0
-ipython==7.8.0
+# ipython 5.9.0 is the last LTS release that supports python 2.7.
```

Comment:
can you instead use the python_version syntax? it'll instead be

ipython==5.9.0; python_version < "3.0"
ipython==7.8.0; python_version > "3.0"

Reply:
We've just turned off 2.7 testing for everything but GAE, so I think we can remove this commented out version of ipython

Reply:
done